core/src/test/java/org/infinispan/expiration/impl/ExpirationSingleFileStoreDistListenerFunctionalTest.java

```diff
@@ -0,0 +1,26 @@
+package org.infinispan.expiration.impl;
+
+import org.infinispan.configuration.cache.CacheMode;
+import org.infinispan.configuration.cache.ConfigurationBuilder;
+import org.infinispan.manager.EmbeddedCacheManager;
+import org.infinispan.test.fwk.TestCacheManagerFactory;
+import org.testng.annotations.Test;
+
+@Test(groups = "functional", testName = "expiration.impl.ExpirationSingleFileStoreDistListenerFunctionalTest")
+public class ExpirationSingleFileStoreDistListenerFunctionalTest extends ExpirationStoreListenerFunctionalTest {
+   @Override
+   protected void configure(ConfigurationBuilder config) {
+      config
+              // We set the wakeup interval to max value.  If we have a store we force the reaper to be enabled, but
+              // for this test we can get false positives if the reaper runs at the same time as a manual expiration
+              // process
```

Comment:
How about "Prevent the reaper from running, reaperEnabled(false) doesn't work when a store is present"?